esti/s3_gateway_test.go

```diff
@@ -54,6 +55,69 @@ func newMinioClient(t *testing.T, getCredentials GetCredentials) *minio.Client {
 	return client
 }
 
+func TestS3UploadToReadOnlyRepoError(t *testing.T) {
+	ctx, _, repo := setupTest(t)
+	defer tearDownTest(repo)
+
+	readOnlyRepo := createReadOnlyRepositoryByName(ctx, t, "tests3uploadobjectdestreadonly")
+	defer deleteRepositoryIfAskedTo(ctx, readOnlyRepo)
+
+	minioClient := newMinioClient(t, credentials.NewStaticV4)
+	// create a file that's larger than any internal buffering minio does
+	const tenMBInBytes = 10 * 1024 * 1024
```

Comment:
Consider
```suggestion
	const tenMega = 10 * 1024 * 1024
```
or
```suggestion
	const tenMibi = 10 * 1024 * 1024
```
The current phrasing reads a bit awkwardly to me, perhaps because I am too much of a physicist: you take 10Mi, multiply it by bytes to get 10MiB, and then divide by bytes to get 10Mi.